### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by VivianWilde**
Added Bug Report, Feature Request, and Task templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Task
```
Comment (na933950) [inline .github/ISSUE_TEMPLATE/task.md:2]: Capitalize please.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
```
Comment (na933950) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Capitalize

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
```
Comment (na933950) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: Capitalize

File: .github/ISSUE_TEMPLATE/task.md
Comment (na933950) [inline .github/ISSUE_TEMPLATE/task.md:1]: Is this meant to be a ticket of sorts? I.e someone makes a task assignment when they find a new task to be completed? If so, we can use the [Project Manager](https://github.com/orgs/PoliticalComputerScience/projects/1) feature and then you can hit "Create an issue" and all three of these should be taken care of. If there markdown files are just here as templates for the issues there than this works.

**Review by na933950 — CHANGES_REQUESTED** [review]
See comments

File: .github/ISSUE_TEMPLATE/task.md
Comment (VivianWilde) [reply .github/ISSUE_TEMPLATE/task.md:1]: These are meant to be templates, so when you create an issue you have an option to specify if it's one of these categories. "Task" specifically is meant to be more a general class of "things to do" that don't come under features or bugs. For instance "refactor/document X", etc.

**Review by na933950 — APPROVED** [review]

**Review by na933950 — APPROVED** [review]

**Pull request merged**